Module 2 and Module 3: New/edited functions and example workflows

wokenny13 opened the pull request:
Create Criteria and Methodology TADA Template
- This PR includes two new Mod 3 vignettes in this PR (I think replacing the old Mod3a vignette with these two new ones would be the best route to take, but left 3a in this repository for now if there are other thoughts to keep it in). These vignettes are a continuation of the ExampleMod2 workflow.
- Added UseAURef to TADA_CreateUseParamRef() as an argument input. The idea is that if a user has provided a list of uses to AU assignments, these uses may have been updated to a list of new ones that we may not be able to retrieve from ATTAINS. 
- This PR includes TADA_CreateMLSummaryRef() and TADA_DefineCriteriaMethodology() as the new functions and final steps in creating the criteria and methodology template in TADA format.
- Please review the two new vignettes that walks through the Module 3 workflows in greater detail as part of this PR.

wokenny13 committed:
Use AU Function Doc Example Update

wokenny13 committed:
Update CriteriaMethods.R

wokenny13 committed:
Function Name Change and handling of iterative Criteria Methods updating

User can choose to edit a reference table and refresh Criteria and Methodology file as an iterative process.

Each table is hierarchical. User must specify which table is being updated through the updateRef argument input.

wokenny13 committed:
Update ATTAINSCrosswalks.R

more descriptive comments on function inputs in TADA_CreateParamRef

wokenny13 committed:
Join a paramRef user supplied by TADA Characteristic Name as an option

handling of Char, fraction and speciation of all combinations if a user has those columns.

wokenny13 committed:
Criteria Summary rolling timestamp draft

summary of data compared to Criteria Methods with a rolling timestamp (draft)

wokenny13 committed:
rolling summary edits

wokenny13 committed:
Criteria Summary - Handling of rolling average summary

wokenny13 committed:
Update CriteriaMethods.R

wokenny13 committed:
rolling mean summary draft

wokenny13 committed:
rolling weighted average dataframe calculation update

wokenny13 committed:
Aggregate data by duration and scatterplot in TADA format test

wokenny13 committed:
Update CriteriaMethods.R

TADA aggregted data frame scatterplot testing

wokenny13 committed:
cleaned up drafted summary output code

wokenny13 committed:
additional summary output updates and minor bug fixes

wokenny13 committed:
Revert "additional summary output updates and minor bug fixes"

This reverts commit feda29ffdabfc28d1a129fe7b6cb3002c3b73cab.

wokenny13 committed:
Update CriteriaMethods.R

wokenny13 committed:
Update CriteriaMethods.R

wokenny13 committed:
Update CriteriaMethods.R

Update R documentation

wokenny13 committed:
Update ATTAINSCrosswalks.R

change siteRef to MLSummaryRef

change sitesAURef to MLAURef

wokenny13 committed:
Update CriteriaMethods.R

sitesAURef changed to MLAURef

wokenny13 committed:
Update CriteriaMethods.R

wokenny13 committed:
variable/function name changes

wokenny13 committed:
Merge branch 'develop' into Create-Module-3-AURef

wokenny13 committed:
Update CriteriaMethods.R

wokenny13 committed:
summary plots enhancements

wokenny13 committed:
Update Criteria Methodology Function

wokenny13 committed:
Merge branch 'develop' into Create-Module-3-AURef

wokenny13 committed:
Revert "use httr2 and update vignettes"

This reverts commit f2a4f11fa201969e4040596b0afa922e0a737b2f.

wokenny13 committed:
Update ATTAINSCrosswalks.R

wokenny13 committed:
Update ATTAINSCrosswalks.R

added match.arg option

wokenny13 committed:
mod 3 vignettes

wokenny13 committed:
Merge branch 'develop' into Create-Module-3-AURef

wokenny13 committed:
Update TADAModule3b.Rmd

wokenny13 committed:
ExampleMod2Workflow Updates

wokenny13 committed:
document text edit

wokenny13 committed:
Updated workflow options for generating Criteria Methods Table

plus edits to the vignette with example workflows

wokenny13 committed:
TADA.ComparableDataIdentifier kept in CriteriaMethods

should we consider this as an arg input? To include or exclude it in future consideration?

wokenny13 committed:
formatting and different arg input combo handling

wokenny13 committed:
arg input function updates to criteria methods

User Supplied + autofill arg input
criteiramethods =
autofill = T/F
displayUniqueId = T/F
updateRef = NONE

Step by step option:
MLSummaryRef =
display = FALSE
autofill = FALSE

wokenny13 committed:
epa304a incorporation to Criteria Table option

wokenny13 committed:
update to mod 3 vignette and workflow options

wokenny13 committed:
ExampleMod3Workflow vignette

Continuation of ExampleMod2Workflow

wokenny13 committed:
Mod 3 workflow and function updates

wokenny13 committed:
useAURef incorporated to useParamRef function

wokenny13 committed:
excel criteria methods allowable value fix

wokenny13 committed:
Update ATTAINSCrosswalks.R

wokenny13 committed:
exmod2workflow considerations (draft)

wokenny13 committed:
Create ExampleMod3CritieriaMethodsAltOptions vignette

wokenny13 committed:
TADA_CreateMLSummaryRef edits

wokenny13 committed:
updates to TADA_CreateMLSummaryRef

wokenny13 committed:
fixed MLSummaryRef bug

wokenny13 committed:
Clean up MLSummaryRef

wokenny13 committed:
.rda example files created, updates to User supplied criteria table bug fix and improvements

wokenny13 committed:
Update CriteriaMethods.R

wokenny13 committed:
Updates to vignettes and criteria function

wokenny13 committed:
Mod3 Vignette Updates

wokenny13 committed:
update wordlist and styler

wokenny13 committed:
Styler, text edits to vignette, moved .rda location

wokenny13 committed:
vignette change excel = FALSE

can't download excel files in checks.

wokenny13 committed:
vignette updates

removed 3b, moved a .rda file to the data folder.

wokenny13 committed:
update Mod3 vignettes

wokenny13 committed:
Update ExampleMod3CriteriaMethodsAltOptions.Rmd

wokenny13 committed:
test-that basic test case and check fixes

utilities global variables, r documentation fixes, and a test-that function.

wokenny13 committed:
dontrun r document example

wokenny13 committed:
R function documentation updates

hillarymarler committed:
Add data used in mod3 vignette to example data and required maintenance

also updated refs in the example mod 3 workflow

hillarymarler committed:
Update ExampleMod3Workflow.Rmd

fix typo when calling example data

wokenny13 committed:
r documentation fixes

missing args documented. minor text fixes.

wokenny13 committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

wokenny13 committed:
run document

fixed missing or undocumented args and run document on it.

added @details to the ExampleData.R due to a message about Level 1 headings are not supported in @format

hillarymarler committed:
fix documentation bugs

hillarymarler committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

hillarymarler committed:
Update ExampleMod3Workflow.Rmd

wokenny13 committed:
empty line break fix in .Rd output

hillarymarler committed:
fixing water type issue in mod 3 workflow

hillarymarler committed:
Update requiredMaintenance.R

hillarymarler committed:
Merge branch '646-legend-for-color-coding-of-wqp-observations-and-what-they-correspond-to-in-tada_viewattains' into Create-Module-3-AURef

hillarymarler committed:
updated example MT data - still need to do a little water type troubleshooting

wokenny13 committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

wokenny13 committed:
Draft Data Dictionary for Criteria and Methods template

Only generated when excel = T

hillarymarler committed:
water type troubleshooting

hillarymarler committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

hillarymarler committed:
geospatial crosswalk updateas

wokenny13 committed:
Data Dictionary Updates

changed from stop to warning to handle the case for when if an org has not submitted data to ATTAINS yet to still run.

hillarymarler committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

wokenny13 committed:
criteria method excel output bug fix

set activesheet, fixes excel output bug

small edits to data dictionary tab

hillarymarler committed:
bug fixes for TADA_CreateAUMLCrosswalk when org_id or au_ref = NULL

hillarymarler committed:
bug fix for au_ref = NULL for TADA_CreateAUMLCrosswalk

hillarymarler committed:
work on internal function for joining crosswalks from different sources

hillarymarler committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

hillarymarler committed:
bug fix for test_mistmatch of ATTAINS water types in TADA_CreateAUMLRef

hillarymarler committed:
updates to rename ATTAINS and org_id for ATTAINS_crosswalk

hillarymarler committed:
TADA_CreateAUMLCrosswalk bug fixes

Includes update so that ATTAINS.OrganizationIdentifier in ATTAINS_crosswalk output is equal to org_Id.

hillarymarler committed:
code cleanup and added comments to TADA_CreateAUMLCrosswalk

added comments, removed intermediate objects, used internal function to make code more concise

hillarymarler committed:
draft of internal function combineATTAINSGeo

commented out within TADA_GetATTAINSByAUID, internal function is not working yet

hillarymarler committed:
added comments and made code more concise for TADA_GetATTAINSByAUID

hillarymarler committed:
correct title/description mismatch in ExampleMod3 vignette

cristinamullin committed:
Merge branch '646-legend-for-color-coding-of-wqp-observations-and-what-they-correspond-to-in-tada_viewattains' into Create-Module-3-AURef

cristinamullin commented on R/GeospatialFunctions.R:
```diff
@@ -2014,34 +2014,42 @@ TADA_GetATTAINSByAUID <- function(.data, au_ref = NULL, add_catch = FALSE) {
 #'
-#' Visualizes the data returned from TADA_CreateAUMLCrosswalk or TADA_CreateATTAINSAUMLCrosswalk if return_sf was set to TRUE.
+#' This function is designed to visualize the data included in the list returned 
+#' from TADA_CreateAUMLCrosswalk. The map can be used to review different 
```
@hillarymarler I updated the description to focus on using TADA_CreateAUMLCrosswalk to simplify the workflow. 

TADA_CreateATTAINSAUMLCrosswalk can be used as well but it may be confusing to most users to include both in the main description. I still included that either can be used in the .data argument.

hillarymarler committed:
fix for legend bug that was causing error for ViewATTAINS in Cybertown.Rmd

hillarymarler committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

hillarymarler committed:
Big fix for TADA_GetATTAINSByAU

wokenny13 committed:
excel output updates

rearranged acutechronic and format fixes to DataDictionary.

wokenny13 committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

hillarymarler committed:
org_id = NULL returns all AU/ML crosswalk matches from ATTAINS in TADA_GetATTAINSAUMLCrosswalk

hillarymarler committed:
Update ATTAINSCrosswalks.R

hillarymarler committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

hillarymarler committed:
TADA_GetATTAINAUMLCrosswalk will return all results from all orgs when org_id = NULL

wokenny13 committed:
criteria methods and autoassign option condensed

if a user supplies their own criteria methods, any remaining non-matches will get assigned from the auto_assign = TRUE option ( should provide a useAURef to reflect any uses that can't be retrieved from ATTAINS as the uses for assignments)

wokenny13 committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

cristinamullin commented on FormattedTable.xlsx:
@wokenny13 was this included here at the top level of the package by accident?

wokenny13 replied:
Yes, this should not be included

cristinamullin commented on myfileRef.xlsx:
@wokenny13 same question for this file. Was this included by accident?

wokenny13 replied:
Yes, this was include by accident as well and should be removed

cristinamullin commented on DESCRIPTION:
```diff
@@ -86,7 +87,7 @@ Imports:
 Depends: 
     R (>= 4.1.0)
 Suggests:
-    curl (>= 7.0.0),
```
I think curl (>= 7.0.0) is required for dataRetrieval to work.

https://github.com/DOI-USGS/dataRetrieval/issues/827

wokenny13 replied:
Likewise, I think this should be changed back to curl (>= 7.0.0)

cristinamullin commented on DESCRIPTION:
```diff
@@ -76,7 +77,7 @@ Imports:
     jsonlite, 
     openxlsx,
     DT,
-    dataRetrieval (>= 2.7.21),
```
Is there a reason the 2.7.21 specification was removed for dataRetrieval? We may need to include both curl and httr2 as dependencies if we do not specify. Right now I just have those two under suggests.

wokenny13 replied:
I did not make updates to the 2.7.21 specification, so I think we could bring this back in

cristinamullin commented on R/DataDiscoveryRetrieval.R:
```diff
@@ -1486,49 +1480,25 @@ make_groups <- function(x, maxrecs) {
+#' # Find web service URLs for each Profile using WQP User Interface (https://www.waterqualitydata.us/)
+#' # Example WQP URL: https://www.waterqualitydata.us/#statecode=US%3A09&characteristicType=Nutrient&startDateLo=04-01-2023&startDateHi=11-01-2023&mimeType=csv&providers=NWIS&providers=STEWARDS&providers=STORET
 #'
-#' # Construct URLs for different profiles
```
@wokenny13 @hillarymarler All of the check notes related to URLs being too long in the documentation should have already been addressed in a previous PR. It looks like this PR may revert those updates unless either of you made additional changes?

wokenny13 replied:
No updates were made on my end for DataDiscoveryRetrieval. Should this .R file be reverted back in?

hillarymarler committed:
TADA_GetATTAINSSUMLCrosswalk works with multiple org_ids

hillarymarler committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

hillarymarler committed:
bug fix for example data

wokenny13 committed:
remove accidental .xlsx file and other mod3 item fixes

criteriaMethods arg input - handling of potential missing columns bug fix

criteriaMethods handling of missing optional columns in a user supply table handling

criteria_table.rda updated table from TT is now reflected

missing call to explicit pakcage name for a function.

cristinamullin committed:
Merge branch 'develop' into Create-Module-3-AURef

hillarymarler committed:
Update GeospatialFunctions.R

hillarymarler committed:
Update GeospatialFunctions.R

wokenny13 committed:
Update CriteriaMethods.R

wokenny13 committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

wokenny13 committed:
autoassign displays all unique Char. Full data frame pull error handling in TADA_CreateMLSummaryRef

When assigning uses and param to sites, if displayNA == TRUE to assign all uses and param to a site regardless if it contains the param, if there are many uses and params, this would cause TADA_CreateMLSUmmaryRef to crash. Limits this option only if the number of sites and uses/param are a reasonable amount (1000 rows or1000 sites are the set limits for doing so). Else defaults to displayNA == FALSE

handling of auto-assign = TRUE to display all WQP char in the output.

include test-that for criteria methods

wokenny13 committed:
edits to texts and example function inputs

wokenny13 committed:
update example data

wokenny13 committed:
updates for subgroup meeting

cristinamullin committed:
do not eval TADACybertown.Rmd

wokenny13 committed:
missing global functions call

also edits to Rd file 'Data_MT_AUMLRef.Rd': to make less than 90 characters in length

cristinamullin committed:
do not eval

wokenny13 commented on R/RequiredCols.R:
```diff
@@ -282,38 +282,18 @@ last.cols <- c(
 #'
 #' @examples
 #' \dontrun{
-#' # Find web service URLs for each Profile using WQP User Interface
```
Should the changes in RequiredCols.R be reverted? My PR should not have made any changes to this file

cristinamullin committed:
Merge branch 'develop' into Create-Module-3-AURef

cristinamullin commented on R/Figures.R:
```diff
@@ -496,10 +496,10 @@ TADA_OverviewMap <- function(.data) {
-          ymin = min(sumdat$TADA.LatitudeMeasure),
-          xmax = max(sumdat$TADA.LongitudeMeasure),
-          ymax = max(sumdat$TADA.LatitudeMeasure)
+          xmin = min(sumdat$TADA.LongitudeMeasure, na.rm = TRUE),
```
double check if na.rm = TRUE should be included here or not

wokenny13 replied:
This is included in Hillary's closed PR

cristinamullin committed:
Create checkColName.Rd

cristinamullin committed:
update tests and vignettes

cristinamullin commented on R/CriteriaMethods.R:
```diff
@@ -0,0 +1,1113 @@
+                                           updateRef = c("none", "paramRef", "useParamRef", "MLSummaryRef"), # hierarchical dependency
+                                           epa304a = FALSE, displayUniqueId = FALSE, excel = TRUE, overwrite = FALSE) {
+  # Excel ref files to be stored in the Downloads folder location.
+  downloads_path <- file.path(Sys.getenv("USERPROFILE"), "Downloads", "myfileRef.xlsx")
```
consider looking for onedrive first

```r
# Define the OneDrive Downloads path
onedrive_downloads_path <- file.path(Sys.getenv("USERPROFILE"), "OneDrive", "Downloads", "myfileRef.xlsx")

# Define the default Downloads path
default_downloads_path <- file.path(Sys.getenv("USERPROFILE"), "Downloads", "myfileRef.xlsx")

# Check if the OneDrive Downloads path exists, and prioritize it
if (file.exists(onedrive_downloads_path)) {
  downloads_path <- onedrive_downloads_path
} else {
  downloads_path <- default_downloads_path
}

# Now downloads_path contains the correct path

```

wokenny13 replied:
would you like for me to make these edits? Prioritizing the OneDrive downloads path seems like a good option to take

cristinamullin committed:
add MT.UseAURef_with_WaterUseRef

Add documentation for and code to reproduce MT.UseAURef_with_WaterUseRef.

Partially addresses check warning: ❯ checking for missing documentation entries ... WARNING
  Undocumented code objects:
    'MT.UseAURef_with_WaterUseRef' 'criteria_table'
  Undocumented data sets:
    'MT.UseAURef_with_WaterUseRef' 'criteria_table'
  All user-level objects in a package should have documentation entries.
  See chapter 'Writing R documentation files' in the 'Writing R
  Extensions' manual.

wokenny13 committed:
address check issues

ExampleData.R documentation updates (re arrange and re formatted some of the roxygen2 inputs)

changed to Data_MT.UseAURef_Water.rda to shorten name

moved criteria_table to inst/extdata

add User.WaterType to globalvariables.

cristinamullin committed:
minor check and documentation updates

cristinamullin committed:
Update TADA_DefineCriteriaMethodology.Rd

cristinamullin committed:
update description, address long urls

address check issues

cristinamullin committed:
Update TADAModule1_AdvancedTraining.Rmd

cristinamullin committed:
Merge branch 'develop' into Create-Module-3-AURef

cristinamullin committed:
fix long urls in examples

cristinamullin commented:
@hillarymarler @wokenny13 what is the status of this issue? https://github.com/USEPA/EPATADA/issues/551. Should the branch associated with this issue be merged into this PR?

wokenny13 committed:
criteria methods function, message output and comment revisions

wokenny13 committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

cristinamullin committed:
Merge branch 'develop' into Create-Module-3-AURef

wokenny13 committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

cristinamullin committed:
update WQXValidationService.Rmd

wokenny13 committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

wokenny13 committed:
remove updateRef param arg in TADA_DefineCriteriaMethodology

In addition: updated Data_Nutrients_UT.rda

wokenny13 committed:
remove updateRef calls from within TADA_DefineCriteriaMethodology

wokenny13 committed:
Update TADAModule1_AdvancedTraining.Rmd

explicit call to load in Data_Nutrient_UT

hillarymarler committed:
Update GeospatialFunctions.R

hillarymarler committed:
Update GeospatialFunctions.R

hillarymarler committed:
bug fix for extra rows with water type NA in ATTAINS_crosswalk output from TADA_CreateAUMLRef

cristinamullin committed:
minor edits

cristinamullin committed:
Module 2 review

fix TADA_GetATTAINSByAUID bug; add error messaging for TADA_ViewATTAINS; update ExampleMod2Workflow.Rmd

cristinamullin committed:
add icons folder back

merge issue

wokenny13 committed:
Update downloads_path to one drive specific downloads if found

cristinamullin committed:
Update ExampleMod2Workflow.Rmd

address merge issue

cristinamullin committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

cristinamullin committed:
Update ExampleMod2Workflow.Rmd

hillarymarler committed:
change ATTAINS colname from ATTAINS.OrganizationId to ATTAINS.OrganizationIdentifier for consistency

based on comment from Ben's review

hillarymarler committed:
Update ExampleMod2Workflow.Rmd

update current limitations/future improvements

cristinamullin committed:
Update ExampleMod2Workflow.Rmd

cristinamullin committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

wokenny13 committed:
Draft test_that for TADA_CreateUseParamRef

Also addresses some minor bug outputs caught while creating test_that for TADA_CreateUseParamRef

hillarymarler committed:
update TADA_CreateAUMLCrosswalk print messages

cristinamullin committed:
minor print message update, vignette updates

cristinamullin committed:
update docs and change default for fill_catchments = FALSE

cristinamullin committed:
update Data_MT_AUMLRef

cristinamullin committed:
combine Module 2 workflows into single vignette

cristinamullin committed:
Update ExampleMod3Workflow.Rmd

cristinamullin committed:
update function input names and vignettes

change fill_catchments to fill_USGS_catch; change add_catch to fill_ATTAINS_catch

cristinamullin committed:
use example data in mod 2 vignette

cristinamullin committed:
adds .TADA_UpdateRefFiles and .TADA_UpdateExampleData

Adding internal maintenance functions and leveraging them in maintenance-scheduled.yaml.

cristinamullin committed:
update refs and example data

cristinamullin committed:
add examples to TADA_CreateAUMLCrosswalk docs

cristinamullin committed:
spellcheck

cristinamullin committed:
vignette update

cristinamullin committed:
Update TADAModule2.Rmd

cristinamullin committed:
update icons

hillarymarler committed:
Update TADAModule2.Rmd

cristinamullin committed:
utils::data

update example data calls to include utils::

hillarymarler committed:
Update TADAModule2.Rmd

cristinamullin committed:
add utils::data

cristinamullin committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

cristinamullin committed:
review new rows in WQXMeasureQualifierCodeRef.csv

update Not Reviewed rows in WQXMeasureQualifierCodeRef.csv

wokenny13 committed:
Update DataDictionary and test-ATTAINSCrosswalk.R

criteria table excel definitions finished defining (needs review still)

explicit call to testthat functions in test

cristinamullin committed:
fix check errors and warnings

cristinamullin committed:
Update Participatory-Science-Water-Projects-in-WQX.Rmd

hillarymarler committed:
tryin viewattains map saved as widget

wokenny13 committed:
test-ATTAINSCrosswalk.R warning fixes

suppresswarnings and other warning message handling for when they are expected warnings.

hillarymarler committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef

hillarymarler committed:
potential bug fix for mod 2 vignette (view attains icons)

wokenny13 committed:
use system.file() for extdata icon images

hillarymarler committed:
icon update

hillarymarler committed:
move icons back to extdata

hillarymarler committed:
Update GeospatialFunctions.R

hillarymarler committed:
bug fix to keep icon backgrounds transparent when knitting mod2 vignette

jbousquin committed:
Update maintenance-scheduled.yaml

Look for maintenance-branch first, use default (develop) if that fails. Still commits to maintenance-branch either way. Checks are triggered when the branch has a PR.

hillarymarler committed:
fix legend bug

cristinamullin committed:
Update maintenance-scheduled.yaml

hillarymarler committed:
Update TADAModule2.Rmd

set return_nearest = FALSE for MT Mod2 example

hillarymarler committed:
mod 2 vignette - set example to return_nearest = TRUE

added text to clarify that in this example return_nearest = TRUE

cristinamullin committed:
Merge branch 'Create-Module-3-AURef' of https://github.com/USEPA/EPATADA into Create-Module-3-AURef
